### ISSUE TITLE
OG UI tests never ran

### ISSUE THREAD

**Issue by claudiu-cristea** [Gizra/og#324]
Just take a look in `web/modules/contrib/og/scripts/travis-ci/run-test.sh` this line:

```bash
./vendor/bin/phpunit -c ./core/phpunit.xml.dist $MODULE_DIR/tests
```

This will run all the tests from the main module but not from the submodule `og_ui`.

Also the test `\Drupal\og_ui\Tests\BundleFormAlterTest`, that never ran on Travis, is failing. It needs this patch to be applied:

```diff
diff --git a/og_ui/tests/src/Functional/BundleFormAlterTest.php b/og_ui/tests/src/Functional/BundleFormAlterTest.php
index dd1ebc28..8f326fcd 100644
--- a/og_ui/tests/src/Functional/BundleFormAlterTest.php
+++ b/og_ui/tests/src/Functional/BundleFormAlterTest.php
@@ -6,7 +6,6 @@
 use Drupal\node\Entity\NodeType;
 use Drupal\og\Og;
 use Drupal\og_ui\BundleFormAlter;
-use Drupal\simpletest\AssertContentTrait;
 use Drupal\Tests\BrowserTestBase;
 
 /**
@@ -16,8 +15,6 @@
  */
 class BundleFormAlterTest extends BrowserTestBase {
 
-  use AssertContentTrait;
-
   /**
    * {@inheritdoc}
    */
``` 

This was discovered while working on #323.

**Comment by amitaibu** [comment]
Thanks @claudiu-cristea !

@AronNovak can you take a look please.

**Comment by MPParsley** [comment]
This has been fixed in #375. A follow up has been created in #377.
